web/src/i18n/locales/en.yaml

```diff
@@ -142,13 +142,13 @@
         subheader: "Brief Activity Summary"
       detail:
         standard:
-          subheader: "Describe the activity and include relevant information."
+          subheader: "There will be opportunities to add aditional text in other sections."
```

Comment:
How about this teak? "This should very brief, one or two sentences describing the activity. There will be opportunities to add additional text in other sections"